.gitignore

```diff
@@ -3,3 +3,5 @@
 
 # Developer Tooling
 .vscode
+
+router
```

Comment:
- A comment to say what this relates to would be useful
- Needs newline at EOF

Reply:
I've removed router for now, we can add this in a separate PR for supergraphs